last_updated_validator/Validator/LastUpdatedValidator.php

```diff
@@ -0,0 +1,189 @@
+<?php
+
+namespace Validator;
```

Comment:
Let's move the whole validator into _plugin folder to avoid a mess in the root directory.